api/src/main/scala/dx/api/DxDatabase.scala

```diff
@@ -0,0 +1,60 @@
+package dx.api
+
+import spray.json._
+
+case class DxDatabaseDescribe(project: String,
+                              id: String,
+                              name: String,
+                              folder: String,
+                              created: Long,
+                              modified: Long,
+                              properties: Option[Map[String, String]],
+                              details: Option[JsValue],
+                              databaseName: String,
+                              uniqueDatabaseName: String)
+    extends DxObjectDescribe
+
+case class DxDatabase(id: String, project: Option[DxProject])(dxApi: DxApi = DxApi.get)
+    extends DxDataObject {
+  def describe(fields: Set[Field.Value] = Set.empty): DxDatabaseDescribe = {
+    val projSpec = DxObject.maybeSpecifyProject(project)
+    val defaultFields =
+      Set(Field.Project, Field.Id, Field.Name, Field.Folder, Field.Created, Field.Modified)
+    val databaseSpecificFields = Set(Field.DatabaseName, Field.UniqueDatabaseName)
+    val allFields = fields ++ defaultFields ++ databaseSpecificFields
+    val descJs =
+      dxApi.databaseDescribe(id, projSpec + ("fields" -> DxObject.requestFields(allFields)))
+    val desc =
+      descJs.getFields("project",
```

Comment:
https://documentation.dnanexus.com/developer/api/introduction-to-data-object-classes/databases
some fields are not included...are we ignoring them for some certain reason?